log/identifier.go

```diff
@@ -24,5 +24,12 @@ import (
 // for this log at distributors, and that will be used to feed
 // checkpoints to witnesses.
 func ID(origin string, key []byte) string {
-	return fmt.Sprintf("%x", sha256.Sum256(key))
+	// Hash both the origin and key, and then hash them together
```

Comment:
I think the comment probably needs removing/updating here, too.

Reply:
Good spot. I thought I'd done that 🤷